File: codes_python/A1-022.py
day = int(input())
month = int(input())
if (month == 12 and day >= 22) or (month == 1 and day <= 19):
    sign = "capricorn"
elif (month == 1 and day >= 20) or (month == 2 and day <= 18):
    sign = "aquarius"
elif (month == 2 and day >= 19) or (month == 3 and day <= 20):
    sign = "pisces"
elif (month == 3 and day >= 21) or (month == 4 and day <= 19):
    sign = "aries"
elif (month == 4 and day >= 20) or (month == 5 and day <= 20):
    sign = "taurus"
elif (month == 5 and day >= 21) or (month == 6 and day <= 21):
    sign = "gemini"
elif (month == 6 and day >= 22) or (month == 7 and day <= 22):
    sign = "cancer"
elif (month == 7 and day >= 23) or (month == 8 and day <= 22):
    sign = "leo"
elif (month == 8 and day >= 23) or (month == 9 and day <= 22):
    sign = "virgo"
elif (month == 9 and day >= 23) or (month == 10 and day <= 23):
    sign = "libra"
elif (month == 10 and day >= 24) or (month == 11 and day <= 21):
    sign = "scorpio"
else:
    sign = "sagittarius"
print(sign)

File: codes_c/A1-022.c
#include <stdio.h>
int main()
{
    int d, m;
    scanf("%d%d", &d, &m);
    if ((m == 12 && d >= 22) || (m == 1 && d <= 19))
        printf("capricorn\n");
    else if ((m == 1 && d >= 20) || (m == 2 && d <= 18))
        printf("aquarius\n");
    else if ((m == 2 && d >= 19) || (m == 3 && d <= 20))
        printf("pisces\n");
    else if ((m == 3 && d >= 21) || (m == 4 && d <= 19))
        printf("aries\n");
    else if ((m == 4 && d >= 20) || (m == 5 && d <= 20))
        printf("taurus\n");
    else if ((m == 5 && d >= 21) || (m == 6 && d <= 21))
        printf("gemini\n");
    else if ((m == 6 && d >= 22) || (m == 7 && d <= 22))
        printf("cancer\n");
    else if ((m == 7 && d >= 23) || (m == 8 && d <= 22))
        printf("leo\n");
    else if ((m == 8 && d >= 23) || (m == 9 && d <= 22))
        printf("virgo\n");
    else if ((m == 9 && d >= 23) || (m == 10 && d <= 23))
        printf("libra\n");
    else if ((m == 10 && d >= 24) || (m == 11 && d <= 21))
        printf("scorpio\n");
    else
        printf("sagittarius\n");
    return 0;
}
